src/any_agent/evaluation/evaluation_case.py

```diff
@@ -9,50 +9,21 @@
 
 class EvaluationCase(BaseModel):
     model_config = ConfigDict(extra="forbid")
-    ground_truth: list[GroundTruthAnswer] = Field(
-        default_factory=list[GroundTruthAnswer],
-    )
+    ground_truth: GroundTruthAnswer | None = None
     checkpoints: list[CheckpointCriteria] = Field(
         default_factory=list[CheckpointCriteria],
     )
     llm_judge: str
-    final_output_criteria: list[CheckpointCriteria] = Field(
-        default_factory=list[CheckpointCriteria],
-    )
 
```

Comment:
I will make some more updates to explain more about what each of these attributes means, and then I will pull that into the documentation so that it is explained there as well.